dnf5daemon-server/dbus/interfaces/org.rpm.dnf.v0.Goal.xml

```diff
@@ -94,6 +94,36 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
         <arg name="options" type="a{sv}" direction="in" />
     </method>
 
+
+    <!--
+        offline_transaction_pending:
+        @pending: boolean, true if there is a pending offline transaction
+
+        Check whether there is an offline transaction configured for the next reboot (i.e. checks the /system-update symlink existence).
+    -->
+    <method name="offline_transaction_pending">
```

Comment:
maybe call it `get_offline_transaction_pending`?

Reply:
Given we are moving to a new Offline interface, I'd call it simply `check_pending()`. What do you think?

Reply:
Sure, sounds good to me. The `get_status` would imply it can be more than just "is/is-not ready".

Reply:
Exactly. Actually we can later provide a call returning all the details about the scheduled transaction.